react-app/src/components/Extract.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { LinkContainer } from 'react-router-bootstrap'
+import { Col, Form, Row } from 'react-bootstrap'
+
+import '../stylesheets/Page.css';
+
+class Extract extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            configPath: "",
+            logPath: "",
+            includeDebug: false,
+            filterStart: false,
+            filterEnd: false
+        }
+
+        this.onConfigFileChange = this.onConfigFileChange.bind;
+        this.onLogFileChange = this.onLogFileChange.bind;
+        this.onIncludeDebugChange = this.onIncludeDebugChange.bind;
+        this.onFilterStartChange = this.onFilterStartChange.bind;
+        this.onFilterEndChange = this.onFilterEndChange.bind;
+        this.onSubmit = this.onSubmit.bind(this);
+    }
+
+    onConfigFileChange(e) {
+        this.setState({
+            configPath: e.target.value
+        });
+    }
+
+    onLogFileChange(e) {
+        this.setState({
+            logPath: e.target.value
+        });
+    }
+
+    onIncludeDebugChange(e) {
+        this.setState({
+            includeDebug: e.target.value
+        });
+    }
+
+    onFilterStartChange(e) {
+        this.setState({
+            filterStart: e.target.value
+        });
+    }
+
+    onFilterEndChange(e) {
+        this.setState({
+            filterEnd: e.target.value
+        });
+    }
+
+    onSubmit(e) {
+        console.log(e);
+        console.log(this.configPath);
+    }
+
+    render() {
+        this.whatever = 10;
```

Comment:
Assuming this variable doesn't get used for anything, can we remove it?

Reply:
This was a placeholder so I could start building the form without getting errors about not using "this" yet. I've deleted it.